### ISSUE TITLE
OmniSharp settings

### ISSUE THREAD

**Issue by deathaxe** [sublimelsp/LSP-OmniSharp#44]
Reading some code from [vscode-csharp](https://github.com/dotnet/vscode-csharp/blob/15e7940baf1fbc2890c3e7f184fdd8330865ca1a/src/omnisharp/server.ts#L414-L542) and [OmniSharp Wiki](https://github.com/OmniSharp/omnisharp-roslyn/wiki/Configuration-Options) it seems language server settings and configuration is handled completely different, than we are used to.

I may be wrong, but those resources read as if all [exposed settings](https://github.com/sublimelsp/LSP-OmniSharp/blob/db4a885fee96a5546852c163dc9a257ad55efdd1/LSP-OmniSharp.sublime-settings#L7-L57) need to be translated to command-line arguments.

At least it seems currently used setting names are not what OmniSharp expects.

**Comment by rwols** [comment]
Good point, and yeah, we should do what the vscode extension is doing there (constructing the startup args with the settings included).

**Comment by deathaxe** [comment]
I don't think, it's useful to follow the VsCode way of handling settings as most supported omnisharp settings are just not supported.

The easy way was to remove them and point to descriptions about how to create global or project-specific omnisharp.json

The hard way was to support real omnisharp settings in dotted form without translating them by any means. We may want to determine those customized by user to only pass required command line arguments.

The `settings` dictionary would look as follows then:

```jsonc
        // https://github.com/OmniSharp/omnisharp-roslyn/wiki/Configuration-Options#msbuild
        "msbuild.enabled": true,
        "msbuild.ToolsVersion": null,
        "msbuild.VisualStudioVersion": null,
        "msbuild.Configuration": null,
        "msbuild.Platform": null,
        "msbuild.EnablePackageAutoRestore" : false,
        "msbuild.MSBuildExtensionsPath": null,
        "msbuild.TargetFrameworkRootPath" : null,
        "msbuild.MSBuildSDKsPath": null,
        "msbuild.RoslynTargetsPath" : null,
        "msbuild.CscToolPath": null,
        "msbuild.CscToolExe": null,
        "msbuild.loadProjectsOnDemand": false,
        "msbuild.GenerateBinaryLogs": false,

        // https://github.com/OmniSharp/omnisharp-roslyn/wiki/Configuration-Options#scripting
        "script.enabled": true,
        "script.defaultTargetFramework": "net461",
        "script.enableScriptNuGetReferences": false,

        "cake.enabled": true,
        "cake.bakeryPath ": null,

        "sdk.path": "/usr/local/share/dotnet/sdk/6.0.403/",
        "sdk.version": "6.0.403",
        // ...
```
